### ISSUE TITLE
Can't translate 'first-child)' pseudo-class

### ISSUE THREAD

**Issue by waja** [mintsoft/check_apache_balancer_members#2]
Since updating to Debian stretch the plugin exits with:

```
Can't translate 'first-child)' pseudo-class at /usr/share/perl5/Web/Scraper.pm line 129.
```

For the checking the new version changes, see:

https://packages.qa.debian.org/p/perl.html
https://packages.qa.debian.org/libw/libweb-scraper-perl.html
https://packages.qa.debian.org/libd/libdata-dump-perl.html
https://packages.qa.debian.org/libm/libmonitoring-plugin-perl.html

Thank, Jan.

**Comment by mintsoft** [comment]
Thanks, I've got that reproduced here

**Comment by waja** [comment]
> Thanks, I've got that reproduced here

Any idea about this? As the version of libweb-scraper-perl (0.38-1) seems to be the same as in debian jessie, it looks like this isn't the origin of this issie.

Thanks, Jan.

**Comment by mintsoft** [comment]
@waja Sorry, I totally forgot about this as our Debian upgrade has been delayed. I think the problem lies with https://metacpan.org/source/CORION/HTML-Selector-XPath-0.23/Changes I'll try and replace the selector with one that's behaving correctly I think

**Comment by mintsoft** [comment]
@waja this should be fixed in: https://github.com/mintsoft/check_apache_balancer_members/commit/8e1ad17578962b35e4a366717c343f580b04bd21 let me know if it isn't

**Comment by waja** [comment]
> @waja this should be fixed in: 8e1ad17 let me know if it isn't

`Balancer Members OK`... looks nice. Many thanks for taking care.
